src/tracker/index.js

```diff
@@ -82,6 +88,18 @@ export default class Tracker {
     });
   }
 
+  async createLabel(params) {
+    try {
+      await this.octokit.rest.issues.createLabel({ ...this.commonParams, ...params });
+    } catch (e) {
+      if (!e.toString().includes('"code":"already_exists"')) {
+        // log error when error is unknown
+        logger.error(`Could not create label ${params.name}`);
+        logger.error(e.toString());
+      }
+    }
+  }
```

Comment:
Suggestion with all previous comments:

```suggestion
  async createLabel(params) {
    return this.octokit.rest.issues.createLabel({ ...this.commonParams, ...params }).catch(error => {
      if (error.toString().includes('"code":"already_exists"')) {
        return;
      }

      logger.error(`Could not create label "${params.name}": ${error.toString()}`);
    });
  }
```